akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -21,8 +21,6 @@ import scala.compat.java8.FutureConverters._
 
 object Flow {
 
-  private[this] val _identity = new javadsl.Flow(scaladsl.Flow[Any])
```

Comment:
Yeah, likely remained here from some ancient version where we added identity in places